chaco/tools/simple_zoom.py

```diff
@@ -647,28 +647,3 @@ def _next_state_pressed(self):
         to the next state. Implements ToolHistoryMixin.
         """
         self._do_zoom()
-
-    ### Persistence ###########################################################
-
-    def __getstate__(self):
-        dont_pickle = [
-            "always_on",
-            "always_on_modifier",
-            "enter_zoom_key",
-            "exit_zoom_key",
-            "minimum_screen_delta",
-            "event_state",
-            "reset_zoom_key",
-            "prev_zoom_key",
-            "next_zoom_key",
```

Comment:
im not sure where these traits were being magically defined

Reply:
me either, this is the only place I was seeing them in the codebase. Perhaps they once existed but had been removed